debian/control

```diff
@@ -2,11 +2,48 @@ Source: openxray
 Section: games
 Priority: optional
 Maintainer: lya Orlov <eagleivg@mail.ru>
-Build-Depends: cmake, build-essential, libglew-dev, libegl1-mesa-dev, libgles2-mesa-dev, libopenal-dev, libcrypto++-dev, libfreeimage-dev, libfreeimageplus-dev, libogg-dev, libtheora-dev, libvorbis-dev, liblzo2-dev, libjpeg-dev, libncurses5-dev, gcc-8, libpcre3-dev, libsdl2-dev, g++-8, debhelper, lsb-release
+Build-Depends:
+ cmake,
+ build-essential,
+ libglew-dev,
+ libegl1-mesa-dev,
+ libgles2-mesa-dev,
+ libopenal-dev,
+ libcrypto++-dev,
+ libfreeimage-dev,
+ libfreeimageplus-dev,
+ libogg-dev,
+ libtheora-dev,
+ libvorbis-dev,
+ liblzo2-dev,
+ libjpeg-dev,
+ libncurses5-dev,
+ gcc-10,
+ libpcre3-dev,
+ libsdl2-dev,
+ g++-10,
+ debhelper,
 Homepage: https://github.com/OpenXRay/xray-16
+Description: X-Ray Engine 1.6 expansion
+ X-Ray Engine sources based on version 1.6.02. The original engine is used in S.T.A.L.K.E.R.: Call of Pripyat game released by GSC Game World.
 
 Package: openxray
 Architecture: any
-Depends: libc6, libsdl2-2.0-0, libopenal1, libcrypto++6, libfreeimage3, libfreeimageplus3, liblua5.1-0, libogg0, libtheora0, libvorbis0a, liblzo2-2, libjpeg-turbo8, libglew2.0
-Description: X-Ray Engine 1.6 expansion
- X-Ray Engine sources based on version 1.6.02. The original engine is used in S.T.A.L.K.E.R.: Call of Pripyat game released by GSC Game World.
+Multi-Arch: foreign
+Depends:
+ ${shlibs:Depends},
+ ${misc:Depends},
+ openxray-data (= ${source:Version}),
+Pre-Depends: ${misc:Pre-Depends}
+Description: ${source:Synopsis}
+ ${source:Extended-Description}
+
+Package: openxray-data
+Architecture: all
+Multi-Arch: foreign
+Depends:
+ ${misc:Depends},
+Description: X-Ray Engine data files (modules, templates and documentation)
+ ${source:Extended-Description}
+ .
+ This package provides openxray architecture independent data files (modules, templates, documentation etc.). Unless you have openxray installed, you probably do not need this package.
```

Comment:
Please end the file with a new line (https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)